webpack/storybook.config.js

```diff
@@ -48,6 +48,11 @@ module.exports = (baseConfig, env) => {
         ]
     };
     config.plugins.push( new ExtractTextPlugin('[name]-[chunkhash].css'));
+    config.plugins.push(new webpack.DefinePlugin({
+        'process.env': {
+            BROWSER: JSON.stringify(true),
+        }
+    }));
```

Comment:
eventually we should share webpack config between storybooks and our main system with a [`makeConfig` like they're doing in faucet](https://github.com/steemit/faucet/blob/master/webpack/makeConfig.js)

Reply:
Yes that looks good.